### PR TITLE
CSS only collapsible menu in small screen.

### PR DESCRIPTION
**Pull request by nottrobin**
Fixes https://github.com/ubuntudesign/vanilla-framework/issues/538

QA
---

Run with vanillaframework.io, check `http://localhost:3000/patterns/navigation/` in small screen size.

Practice collapsing and expanding the menu until you've got it down.

### CODE REVIEW

File: scss/_patterns_navigation.scss [hunk 1]
```diff
@@ -1,25 +1,80 @@
...
+        width: auto;
+      }
+
       &-link {
         border-bottom: 0;
       }
     }
 
     &__links {
+      // display: none;
```
Comment (deadlight) [inline scss/_patterns_navigation.scss]: Remove, please

**Comment by deadlight** [comment]
Comment in code - just some errant commented code.

Implement it in the actual site nav, why not?!

I'll check up once you've done so.

**Comment by nottrobin** [comment]
@deadlight comment removed. Also updated the example code.

However, implementing it in the site can't be done in this codebase, it needs to be done over at [vanillaframework.io](https://github.com/ubuntudesign/vanillaframework.io).

I've created an [issue over there](https://github.com/ubuntudesign/vanillaframework.io/issues/49).

**Comment by deadlight** [comment]
Looks great.

**Pull request merged**